ignite/pkg/xexec/xexec.go

```diff
@@ -0,0 +1,68 @@
+package xexec
+
```

Comment:
Do we need to export all of these functions?

Reply:
Yes, except maybe the `IsExec` which I left it public because it could be useful, but I can make it private to simplify the package API.

Reply:
It is worth mentioning that the `ResolveAbsPath` is implemented to "replace" `exec.LookPath` to support searching of binaries inside the Go binary directory when it is not in the `$PATH`.

The `TryResolveAbsPath` was added after to fix an issue with the integrations tests and it's used to resolve the absolute path when running chain commands which is the what fixes the reported issue. This is a new implementation to support context where the Go binary path is not added to the system's binary path.